### PR TITLE
fix: combinatives weight into TOPSIS

### PR DESCRIPTION
**Pull request by storyofhis**

### CODE REVIEW

File: src/app/siswa/rekomendasi/page.tsx [hunk 1]
```diff
@@ -1,24 +1,42 @@
...
   type TopsisRecommendation,
   type AhpRecommendation,
   type TopsisAHPRecommendation,
+  type TOPSISCombinativesRecommendation,
 } from "@/types/data/recommendation";
-import { TopsisCard } from "./_components/topsis-card";
-
-export const metadata: Metadata = {
-  title: "Hasil Rekomendasi",
-};
```
Comment (albugowy15) [inline src/app/siswa/rekomendasi/page.tsx:16]: Kenapa metadata nya dihapus?

File: src/app/siswa/rekomendasi/page.tsx [hunk 2]
```diff
@@ -1,24 +1,42 @@
...
+  facilities: Float32Array;
+  total_open_jobs: Float32Array;
+  salaries: Float32Array;
+  entrepreneur_opportunities: Float32Array;
+}
+
+export interface CriteriaWeights {
+  entropy: Weight;
+  ahp: Weight;
 }
```
Comment (albugowy15) [inline src/app/siswa/rekomendasi/page.tsx:40]: Type declaration kayak gini bisa tolong taruh di direktori `types/data`

File: src/app/siswa/rekomendasi/_components/metadata.tsx
Comment (albugowy15) [inline src/app/siswa/rekomendasi/_components/metadata.tsx:1]: Ini buat apa?

**Review by albugowy15 — CHANGES_REQUESTED** [review]

File: src/app/siswa/rekomendasi/page.tsx (hunk 2 above)
Comment (storyofhis) [reply src/app/siswa/rekomendasi/page.tsx:40]: DONE!!

**Review by albugowy15 — APPROVED** [review]

**Pull request merged**